presto-spark-common/src/main/java/com/facebook/presto/spark/PrestoSparkQueryStatusInfo.java

```diff
@@ -54,15 +54,13 @@ public PrestoSparkQueryStatusInfo(
             @JsonProperty("stats") StatementStats stats,
             @JsonProperty("error") Optional<QueryError> error,
             @JsonProperty("warnings") List<PrestoWarning> warnings,
-            @JsonProperty("updateType") Optional<UpdateInfo> updateInfo,
```

Comment:
The change to the JsonProperty was missed here. The name is still updateType.
But we don't really need this property for PrestoSpark* subsystem.